.github/workflows/ci-actions-incremental.yml

```diff
@@ -33,6 +33,10 @@ on:
       - '.github/*.conf'
   workflow_dispatch:
 
+concurrency:
+  group: "workflow = ${{ github.workflow }} && ref = ${{ github.event.ref }} && pr = ${{ github.event.pull_request.id }}"
```

Comment:
I'm surprised you make the group a boolean. In the examples they give, they make group a string and unicity is checked at the string level?

Reply:
It's a string. I can replace `&&` with something else if it's confusing.

Reply:
Done. I replaced ` && ` with `, `.